### PR TITLE
[DE-395] Save unlayer model in doppler

### PR DESCRIPTION
**Pull request by Cromeror**
This implementation change the getCampaign return used in [DE-400](https://makingsense.atlassian.net/browse/DE-400)

### CODE REVIEW

File: Doppler.HtmlEditorApi.Test/GetCampaignTest.cs [hunk 1]
```diff
@@ -121,12 +123,15 @@ public async Task GET_campaign_should_accept_right_tokens_and_return_404_when_DB
         public async Task GET_campaign_should_accept_right_tokens_and_return_content_as_it_is_in_DB(string url, string token, string expectedAccountName, int expectedIdCampaign)
         {
             // Arrange
-            var dbContent = "{ \"hi\": \"this is an Unlayer's design!\" }";
+            Model.ContentModel emptyContentModel = new Model.ContentModel()
```
Comment (ms-darianbenito) [inline Doppler.HtmlEditorApi.Test/GetCampaignTest.cs:126]: Why are you using empty as a prefix?
```suggestion
            Model.ContentModel contentModel = new Model.ContentModel()
```

Comment (Cromeror) [reply Doppler.HtmlEditorApi.Test/GetCampaignTest.cs]: Its ok now!

File: Doppler.HtmlEditorApi.Test/GetCampaignTest.cs [hunk 2]
```diff
@@ -121,12 +123,15 @@ public async Task GET_campaign_should_accept_right_tokens_and_return_404_when_DB
         public async Task GET_campaign_should_accept_right_tokens_and_return_content_as_it_is_in_DB(string url, string token, string expectedAccountName, int expectedIdCampaign)
         {
             // Arrange
-            var dbContent = "{ \"hi\": \"this is an Unlayer's design!\" }";
+            Model.ContentModel contentModel = new Model.ContentModel()
```
Comment (ms-darianbenito) [inline Doppler.HtmlEditorApi.Test/GetCampaignTest.cs:126]: Minor 
```suggestion
            var contentModel = new ContentModel()
```
if my suggestion is accepted, you will need to include the `Doppler.HtmlEditorApi.Model` namespace

Comment (Cromeror) [reply Doppler.HtmlEditorApi.Test/GetCampaignTest.cs]: Yes, I included the namespace `Doppler.HtmlEditorApi.Model`

**Pull request merged**